packages.txt

```diff
@@ -14,4 +14,5 @@ libpq-dev
 libproj-dev
 libsqlite3-mod-spatialite
 pkg-config
+tcl-dev
```

Comment:
This requires re-building `base` first otherwise you'll run into `/bin/sh: 1: tclsh: not found`.

Reply:
Yep – I wasn't sure what to do about it. Adding a note in the README seems redundant if it's only needed once if you've already built the image before.

Although after refactoring it to the multi-stage build, this is no longer the case as they're now two separate images (unless we use the optimisation trick I described in https://github.com/django/django-docker-box/pull/50#discussion_r1957378535).

Reply:
The distinct image approach solved the issue for me so I think we can stick to it without the optimization for now.

Having it main for Django Con Europe sprints might be handy.